samples/benchmarks/coremark/src/main.c

```diff
@@ -49,13 +49,16 @@ static const struct gpio_dt_spec status_led;
 static K_SEM_DEFINE(start_coremark, 0, 1);
 static atomic_t coremark_in_progress;
 
+/* Import the clock frequency to be able to print it. */
+extern uint32_t SystemCoreClock;
```

Comment:
How did it work previously?

Reply:
It must have been declared in some header for other platforms, but I haven't investigated it in details.

Reply:
SystemCoreClock is part of the MDK and is defined as extern inside the system_nrf header. Could you verify if the build works when you include this header?

#include <system_nrf.h>